Client/Frontend/Settings/SettingsTableViewController.swift

```diff
@@ -343,7 +350,12 @@ private class VersionSetting : Setting {
         let buildNumber = NSBundle.mainBundle().objectForInfoDictionaryKey("CFBundleVersion") as! String
         return NSAttributedString(string: String(format: NSLocalizedString("Version %@ (%@)", comment: "Version number of Firefox shown in settings"), appVersion, buildNumber), attributes: [NSForegroundColorAttributeName: UIConstants.TableViewRowTextColor])
     }
-
+	
```

Comment:
Same here (see [whitespace tips in the README](https://github.com/mozilla/firefox-ios#whitespace) for fixing/avoiding this).